cli/integration_tests/run_logging/log_prefix.t

```diff
@@ -31,23 +31,23 @@ Setup
   \xe2\x80\xa2 Packages in scope: app-a (esc)
   \xe2\x80\xa2 Running build in 1 packages (esc)
   \xe2\x80\xa2 Remote caching disabled (esc)
-  cache miss, executing 3df2c74b2bfbc724
+  cache hit, replaying output 31ed90d641a7a6e0
```

Comment:
I think this was a bug in the test previously because `.turbo` wasn't in the `.gitignore` file? Even the comment above this test says it should have been a cache hit. (I wrote the test, so I might have missed it in a rush to get this out)